org.eclipse.ui.editors/forceQualifierUpdate.txt

```diff
@@ -1,3 +1,4 @@
 # To force a version qualifier update add the bug here
 Bug 509931: Comparator errors in M20170104-0545
-Bug 534597 - Unanticipated comparator errors in I20180511-2000
\ No newline at end of file
+Bug 534597 - Unanticipated comparator errors in I20180511-2000
+Bug 26765 - Add linespace support
```

Comment:
This is not necessary as there are some changes in git. They are only useful when jar content change without source changing (eg newer Java target, or different compiler options)